app/views/AppealConfirmationPage.scala.html

```diff
@@ -99,7 +99,7 @@ <h2 class="govuk-heading-m">@messages("appealConfirmation.whatHappensNext")</h2>
     </ul>
 
     <p class="govuk-!-padding-top-4 govuk-body-s" id="feedback-link">
-        <a href="@{appConfig.feedbackUrl}" class="govuk-link">@messages("appealConfirmation.feedbackLink.1").trim</a> @messages("appealConfirmation.feedbackLink.2")
+        <a href="@{appConfig.feedbackUrl(userRequest.uri)}" class="govuk-link">@messages("appealConfirmation.feedbackLink.1").trim</a> @messages("appealConfirmation.feedbackLink.2")
```

Comment:
This url needs to remain as it was as it goes to a different survey form from the one used in the Beta banner. May mean you have to name your new link slightly differently in the config